provisioning/Samples/service/GroupCertificateVerificationSample/GroupCertificateVerificationSample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
For whatever reason, this .sln file didn't seem to exist in the samples repo before so my machine automatically added it.

Reply:
You can delete it. We want to remove all the samples too.